system_status/Dockerfile

```diff
@@ -1,3 +1,16 @@
-FROM public.ecr.aws/lambda/python:3.9
+# Docker image for the SNS to SQS SMS callback the lambda API.
+
+FROM public.ecr.aws/lambda/python:3.10
 
 ENV PYTHONDONTWRITEBYTECODE 1
+
+RUN yum -y install git
```

Comment:
do we need this now?

Reply:
Because we use a [git repo for notification utils](https://github.com/cds-snc/notification-lambdas/pull/47/files/cfa3437d4af7e4ad93edb44a9f210202ed876ca2#diff-6f83115693be0cbfd913d884c0746cba45f91b98d24a04fd8398e97c014d8bcdR4) in the `requirements` file, the container needed to have git to be able to install it!